src/components/Components/ComponentDetails/tabs/ComponentDetails.tsx

```diff
@@ -8,17 +8,31 @@ import {
   FlexItem,
 } from '@patternfly/react-core';
 import yamlParser from 'js-yaml';
+import { useLatestSuccessfulBuildPipelineRunForComponent } from '../../../../hooks/usePipelineRuns';
 import ExternalLink from '../../../../shared/components/links/ExternalLink';
 import { ComponentKind } from '../../../../types';
+import { getPipelineRunStatusResults } from '../../../../utils/pipeline-utils';
+import { useWorkspaceInfo } from '../../../../utils/workspace-context-utils';
 import GitRepoLink from '../../../GitLink/GitRepoLink';
 
 type ComponentDetailsProps = {
   component: ComponentKind;
 };
 
+const RESULT_NAME = 'IMAGE_URL';
+
 const ComponentDetails: React.FC<React.PropsWithChildren<ComponentDetailsProps>> = ({
   component,
 }) => {
+  const { namespace } = useWorkspaceInfo();
+  const [pipelineRun, pipelineRunLoaded, error] = useLatestSuccessfulBuildPipelineRunForComponent(
+    namespace,
+    component.metadata.name,
+  );
```

Comment:
We need to use the latest push build here. If a PR build is the latest, this hook will return that

Reply:
updated